types/soap/index.d.ts

```diff
@@ -1,87 +1,188 @@
-// Type definitions for soap 0.18
+// Type definitions for soap 0.19
 // Project: https://www.npmjs.com/package/soap
-// Definitions by: Leo Liang <https://github.com/aleung>, Cage Fox <https://github.com/cagefox>
+// Definitions by:  Leo Liang <https://github.com/aleung>
+//                  Cage Fox <https://github.com/cagefox>
+//                  Onur Yildirim <https://github.com/onury>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="node" />
 
-import * as events from 'events';
+import { EventEmitter } from 'events';
 
-interface Security {
+export interface ISOAPMethod {
```

Comment:
I don't think there is a consistent style in [node-soap](https://github.com/vpulim/node-soap) project. 

You can also find these:

* `wsdl_headers`
* `wsdl_options`
* `wsdlOptions`
* `addSoapHeader`
* `changeSoapHeader`
* `soapError`

Treating abbreviations as normal words increases readability. For example, a glance at `ISOAPMethod` might become `ISO-AP-Method` or `IS-OAP-Method`.